config/serverless.security.yml

```diff
@@ -8,6 +8,7 @@ xpack.uptime.enabled: false
 
 ## Enable the Serverless Security plugin
 xpack.serverless.security.enabled: true
+xpack.serverless.security.productLineIds: ['securityComplete']
```

Comment:
@semd, @machadoum I know it's kind of a late ask, but maybe it will be more clear to have the next format:
```
xpack.serverless.security.productTypes: [
{product_line: security, product_tier: complete}
]
```

WDYT?

Reply:
You will need to combine it anyway, when transform the config to array, but I was thinking from the extensibility perspective, maybe new product type dimension will be added and we need to treat it differently then just combined id. My comment is not critical, but maybe it will be better to understand for others what we mean by products as well.

Reply:
Yes, I like it! we can use this format, it implies changes in the PLIs config though, we'll have to define them like this:
```
{
  security: {
    essential: [],
    complete: [
      AppFeatureKey.advancedInsights, 
      AppFeatureKey.casesConnectors,
    ],
  },
  endpoint: {
    essential: [...],
    complete: [...],
  },
  cloud: {
    essential: [...],
    complete: [...],
  },
}
```
We'll also have to change the `getProductAppFeatures` function to use this structure. It would also be nice to encapsulate the logic to include all the "essential" PLIs if the tier is "complete", inside this function as well.

@YulNaumenko Do you think we need to apply these changes now, or can we do this change in a follow-up PR? I would also like to change the "enabled PLIs" structure format in which the different plugins communicate, we are using an object with only the `true` values, it would be better to use an array.

Reply:
I don't have any preferences. If it is very complex change, we can do a follow up PR.